bcipy/gui/gui_main.py

```diff
@@ -54,9 +55,11 @@ def add_button(self, message: str, position: tuple, size: tuple,
 
         self.buttons.append(btn)
 
-    def bind_action(self, action: str, btn: wx.Button) -> None:
+    def bind_action(self, action: str, btn) -> None:
         if action == 'default':
             self.Bind(wx.EVT_BUTTON, self.on_clicked, btn)
+        if action == 'default' and isinstance(btn, wx.ComboBox):
```

Comment:
This should come first, then the other if statement button (to avoid binding 2x). add some comments here